### DISCUSSION TITLE
v1.2.2

### DISCUSSION THREAD

**Discussion by zippity21** [zippity21/Stardew_Valley_Regression_Mod#36] [Announcements]
## What's Changed
* Zippity21/larger sprites by @zippity21 in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/33
* Zippity21/underwear durability by @zippity21 in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/35

**Full Changelog**: https://github.com/zippity21/Stardew_Valley_Regression_Mod/compare/v1.2.1...v1.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/zippity21/Stardew_Valley_Regression_Mod/releases/tag/v1.2.2'>v1.2.2</a>.</em>

**Comment by Kerie70** [comment]
The new update is looking pretty nice though I think the fill rate on bladder/bowels still needs to be adjusted some more.  I've noticed especially when you're fully incontinent the rate fills up so fast you can easily end up blowing through 8+ diapers per day if you're actively fishing, mining, etc.  The passive rate on bowels seems to still be pretty high in that it constantly passes bladder by a decent margin.

**Reply by zippity21** [reply]
You know what it probably is... I don't think I thought about what the average amount of work is done in a day. I have it such that doing nothing causes the average number of toilet usages per day, when really you should be going an average amount only if doing an average amount of work, and less if doing nothing.

Perhaps I could use stamina as a reference point. Maybe call using 2/3 of maximum stamina an average" day of of work.

Thoughts?

**Reply by Kerie70** [reply]
If that would be easier for you then using stamina as reference could work.  I think I personally would just drastically lower the value of fill per action by what the average passive tick is and adding that tick plus 0.03-0.05% per action taken.  Right now it seems I can cast my fishing line and every 4-5 times I do it, accident.  2-3 accidents later and I have to change.  Which is slightly annoying cause I made those pink diapers with the most absorbency out of all of the rest. 

Now that I think about it, aside from newer players, an "average" day of work for more vet/skilled players is going to be several bars.  I personally will fish from the moment I get up to the moment I have to sleep and blow through several stamina bars eating the fish to recoup what I use. 

I think what you have as base right now is fine for the passive rate but maybe starting small with a 0.03-0.05% capped increase per tick when performing an action would be a good starting point and then adjusting from there after trying it out in game.

**Reply by zippity21** [reply]
With my current changes (not pushed yet), I have it such that at full continence, you go pee about 3 times per day and poo ~1.5. I updated the underwear so that the best diaper (heart) can handle just over three full (largest) bladders and 1.5 full (largest) bowels. While the rate of bladder/bowel increase changes, the total amount doesn't, so 1 heart diaper should still last a full day.

Of course overeating (eating when not hungry) or over drinking (drinking when not thirsty) overflows into bowel/bladder. So this will effect total amounts. Given the need to eat/drink to restore stamina (and not just hunger and thirst) I think the values need to change so that you can eat enough to almost fill stamina without over-filling hunger. I need to figure this out.

At minimum continence bladder fills at ~1 wetting per hour, and ~1 mess per 2 hours. expending stamina constantly (I tested by using the hoe), the rate about doubles.

Thoughts on this?

**Reply by zippity21** [reply]
Also, I understand the need to eat/drink to refill stamina, so I don't want the hunger/thirst bars to go up to quickly. In the same token, I don't want them to go up too slowly, otherwise it becomes very expensive to keep yourself fed.

I need some input from experienced players as to what is reasonable.
